### PR TITLE
#CX-7913 : Notes Widget Header Integration

### PR DESCRIPTION
**Pull request by rathnavel**
https://jira-eng-sjc12.cisco.com/jira/browse/CX-7913

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Please check if the PR fulfills these requirements
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

#### (Bug fix, feature, docs update, ...)

`[Type here]`

#### (You can link to an open issue)

`[Type here]`

#### (if this is a feature change)?

`[Type here]`

#### (What changes might users need to make in their application due to this PR?)

`[Type here]`

## Other information:

`[Type here]`

### CODE REVIEW

**Review by parsoman — APPROVED** [review]

File: Examples/Widgets/Notes/src/notesWidget.ts [hunk 1]
```diff
@@ -0,0 +1,344 @@
+/**
```
Comment (sbudhia) [inline Examples/Widgets/Notes/src/notesWidget.ts:1]: Can you rename this file and capitalize the first letter?

File: Examples/Widgets/Notes/src/notesWidget.ts [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+    protected updated(changedProperties: PropertyValues): void {
+      super.updated(changedProperties);
+      if (changedProperties.has("agentId")) {
+        this.updateAgentId();
+      }
+    }
+
+    protected firstUpdated(changedProperties: PropertyValues): void {
+      super.firstUpdated(changedProperties);
+      // SERVICE.telemetry.track(SERVICE.telemetry.MIX_EVENT.NOTES_WIDGET_OPENED);
```
Comment (sbudhia) [inline Examples/Widgets/Notes/src/notesWidget.ts:71]: Dont leave thsi in comments...remove it completely

Comment (rathnavel) [reply Examples/Widgets/Notes/src/notesWidget.ts]: > Dont leave thsi in comments...remove it completely

Changes Updated.

**Comment by sbudhia** [comment]
The filename still is in lowercase alphabet

File: Examples/Widgets/Notes/src/notesWidget.ts (hunk 1 above)
Comment (rathnavel) [reply Examples/Widgets/Notes/src/notesWidget.ts]: > Can you rename this file and capitalize the first letter?

File name updated.

**Review by sbudhia — APPROVED** [review]

**Review by sameyada — APPROVED** [review]

**Pull request merged**